src/components/Game/Game.js

```diff
@@ -56,6 +57,30 @@ class GameView extends Component {
         window.removeEventListener('resize', this.debouncedResizeListener);
     }
 
+    handleResize = () => {
+        const { letters, lettersInSlots, dragDisabled } = this.state;
+        if (isMobile() === dragDisabled) {
```

Comment:
this condition bothers me a little bit
basically it means `if "mobileness" has changed` or `if drag disable status has changed`
but here it's mixed (mobile and dragDisabled used at the same time), I had to spend a moment to parse it and I had to remember how `dragDisabled` is computed to figure this out

so maybe it would be better to have sth like:
```
const dragDisabled = …
if (dragDisabled !== state.dragDisabled) {
…
``` 
I believe the condition I suggested is closer to the actual logic that forces us to do state update

Reply:
it bothers me a little bit too. BUT:
this condition means `if screen resolution at the end of resizing is the same (i.e. mob/desk) as it was when resizing started... (do nothing)`. 
this way i want to decide whether the state needs to be changed. and i'm not sure if i can achieve this some other way (the piece of code you wrote doesn't answer my doubts).

Reply:
ooops, I just noticed I was commenting on `isMobile() === dragDisabled` but in my head I was thinking of `isMobile() !== dragDisabled`, so negation is missing in couple of places in my above comment

but I believe my point is still valid, the condition `isMobile() === dragDisabled` is confusing or difficult to understand because you have to remember how `dragDisabled` is calculated in order to understand the condition

perhaps for you it's obvious `dragDisabled` is based on mobile/desktop, it's a hidden assumption of this condition

so fixing my negation problem I would suggest:
```
const dragDisabled = this.getDragDisabled(); // or sth like this
// the condition below is straightforward
// it's comparing the new dragDisabled with the one stored in state (the old one)
// so the condition is true if "drag disable status has not **changed"** (new === old)
if (dragDisabled === state.dragDisabled) { 
…
```

Reply:
also, something I didn't mention above…
now the structure is:
```
if (A) {
  return;
}
if (B) {
  // do X
} else {
  // do Y
}
```
in my opinion this structure works well if `A` is checking for error or loading state (e.g. `if (error) {…`)
but here it's not the case, here you need to remember about `A` to properly understand `B` condition
I feel it would be better to negate `A` and wrap the second `if (B)` inside the block corresponding to `not A`
please consider it as my suggestion, feel free to leave it as it is, I wouldn't be surprised to hear the opposite opinion on that matter

Reply:
> ```
> const dragDisabled = this.getDragDisabled(); // or sth like this
> ...
> if (dragDisabled === state.dragDisabled) { 
> …
> ```

So what you are suggesting is creating a method (`getDragDisabled()` in your example) which would return `isMobile()`?

Reply:
yes, I think it would be cleaner/better

also you can think of it this way: "now drag disabled status HAPPENS to be determined by mobile/desktop but in general it doesn't have to be so don't hardcode it, let's define this logic in one place"

and as far as I remember very similar method already exists (but it invokes setState now, this can be changed), the issue seems to be related to the other discussion below (about setting initial dragDisabled value)